ibc_data/contrasts/PreferenceFaces.csv

```diff
@@ -0,0 +1,4 @@
+condition

```

Comment:
The Preference tasks (faces, food, houses,paintings) is a bit confusing when creating these files because I am not quite sure on how to express those constant, linear and quadratic effects.

This same applies for VSTM and Enumeration tasks.

Reply:
These contrasts are identical to the regressors used.

Reply:
Okay! So I added the contrast lines with the same names and added **1** to each row. Is this what you meant??